### DISCUSSION TITLE
ListBox: Why does selecting an already selected option deselect it?

### DISCUSSION THREAD

**Discussion by HppZ** [primefaces/community#2568] [PrimeVue]
How to disable this behavior?

**Comment by vterzic** [comment]
Hacky, but it works
```
watch(selectedTest, (newValue, oldValue) => {
    if (!newValue) {
        nextTick(() => {
            selectedTest.value = oldValue;
        });
    }
});
```

**Reply by erdemsusam** [reply]
> Hacky, but it works
> 
> ```
> watch(selectedTest, (newValue, oldValue) => {
>     if (!newValue) {
>         nextTick(() => {
>             selectedTest.value = oldValue;
>         });
>     }
> });
> ```

This works but be careful, any other code inside watch works two times because watch triggers again after assigning selectedTest with oldValue.
